packages/js/src/decorator/gutenberg.js

```diff
@@ -285,7 +285,7 @@ function removeAllAnnotationsFromBlock( blockClientId ) {
  * @returns {void}
  */
 export function reapplyAnnotationsForSelectedBlock() {
-	const block = select( "core/editor" ).getSelectedBlock();
+	const block = select( "core/block-editor" ).getSelectedBlock();
```

Comment:
Don't we need to smoke test highlighting in Block editor for this change? 🤔

Reply:
You could be doing that when you're testing this PR, if you, like me, forget step 8 of the testing instructions in the linked issue 😉 

So yeah, please do a little bit smoke testing, in this case, specifically, changing text in a block that has annotation while having the eye marker active. Thanks 🙏